### ISSUE TITLE
Missing header file when trying to compile in Visual Studio

### ISSUE THREAD

**Issue by berserkingyadis** [serpentiem/ddmk#91]
Hey, I wanted to try out compiling this project on my computer and everything appeared to go fine with 1 Error in the "Mary" Sub-Project (sadly the one Im interested in).

I cloned the project (and imgui submodule) and opened the "ddmk.sln" with Visual Studio and hit "Build Solution"

```
========== Build: 3 succeeded, 1 failed, 0 up-to-date, 1 skipped ==========
```

I am on Windows 10 using Visual Studio Community 2017 Version 15.9.18

This is the error:

```
2>Eva.cpp
5>Hooks.cpp
3>Object.cpp
2>Arcade.cpp
3>e:\dev\github\others\ddmk\mary\system\file.h(5): fatal error C1083: Cannot open include file: '../../Zip/zip.h': No such file or directory
3>Done building project "Mary.vcxproj" -- FAILED.
5>Hotkeys.cpp
2>BossRush.cpp
```
According to the error, there is a missing header file `zip.h` in a folder Zip which both arent present in the repository. Did I forget to set up something? 

Any help would be greatly appretiated, Im sure I just missed something small.

PS: Ive tried to send you multiple chat messages on reddit about this project over the last years but you seem to have muted the chat system. Where can I reach you if I have some questions about helping out with this project?

**Comment by serpentiem** [comment]
My bad. Added the missing dependencies. The project should compile fine now.

I indeed block chat requests on reddit, but I still receive messages.

When I'm coding I'm very focussed and ignore distractions. Coupled with the fact that I rarely check my mails this creates long processing times.

I changed my setup to now also receive e-mails upon new messages both on GitHub and reddit.

I'll try to get rid of this bad habbit and answer messages more quickly.

Cheers
